packages/fast-development-site-react/app/app.tsx

```diff
@@ -71,6 +71,7 @@ export default class App extends React.Component<{}, IAppState> {
                 themes={themes}
                 onUpdateTheme={this.handleUpdateTheme}
                 showComponentStatus={true}
+                showTransparencyToggle={true}
```

Comment:
can you add this to the other implementations of the dev-site for `fast-components-react-base` and `fast-components-react-msft`

Reply:
@janechu  Even if it defaults to true?

Reply:
@janechu done